QUEST.tsv

```diff
@@ -3345,7 +3345,7 @@ QUEST_20151224_003352	Paladins are fearless because we believe the goddesses wil
 QUEST_20151224_003353	Now it's turn to test the techniques you've learned so far.{nl}Go and defeat the monsters in Apsimesti Crossroads.
 QUEST_20151224_003354	They say everything works out if you just devote yourself, right?{nl}Orsha and Fedimian are a bit far away but... please think of it as devotion.
 QUEST_20151224_003355	Ardel's Diary Page 1
-QUEST_20151224_003356	Date: XX/XX. Clear sky.{nl}-{nl}Things at the Delmore Fortress are busy as always.{nl}The walls destroyed during Medzio Diena are being rebuilt and everyone is returning to their normal lives.{nl}I  hope nothing like Medzio Diena happens ever again.{nl}Thanks to Lord Delmore I think the fortress will be safe now.
+QUEST_20151224_003356	Date: XX/XX. Clear sky.{nl}-{nl}Things at the Delmore Castle are busy as always.{nl}The walls destroyed during Medzio Diena are being rebuilt and everyone is returning to their normal lives.{nl}I  hope nothing like Medzio Diena happens ever again.{nl}Thanks to Lord Delmore I think the fortress will be safe now.
```

Comment:
There's two spaces between the I and hope in "I  hope nothing like Medzio Diena happens ever again."